Kickstarter-iOS/Views/Controllers/CreatePasswordViewController.swift

```diff
@@ -93,8 +187,57 @@ final class CreatePasswordViewController: UITableViewController {
     guard let footerView = tableView.dequeueReusableHeaderFooterView(withClass: className)
       as? SettingsGroupedFooterView else { return nil }
 
-    footerView.configure(with: Strings.Password_min_length_message())
+    footerView.configure(with: self.viewModel.outputs.currentValidationLabelText())
```

Comment:
In case that the font size gets changed the `tableView` will be reloaded...this is why we have to update footer with the latest value from the `VM` that we can access from a mutable property using `currentValidationLabelText`